src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -122,11 +124,9 @@ public static final class SwerveModuleConstants {
   }
 
   public static final class IntakeConstants {
+    // Pretend it doesn't exist
```

Comment:
This should be a TODO still if the sensor doesn't exist. But also Port0 is a valid port number.